guide/daily-spending-wallet/privacy.md

```diff
@@ -45,222 +46,101 @@ This page should inform about what information is made public when sending or re
-
-{% include image.html
-   image = "/assets/images/guide/daily-spending-wallet/privacy/coin-join.jpg"
-   retina = "/assets/images/guide/daily-spending-wallet/privacy/coin-join@2x.jpg"
-   alt-text = "Example CoinJoin transaction with 3 participants with 1 input each, which get broken up into 5 equal outputs."
-   width = 400
-   height = 400
-   layout = "float-right-desktop"
-   caption = "CoinJoin transactions attempt to make payments more private by mixing inputs from many senders and outputs to many receivers."
-%}
-
-A [CoinJoin]({{ "/guide/glossary/#coinjoin" | relative_url }}) is an advanced technique where multiple participants collaborate on a transaction to break the "common input ownership" heuristic[^3], which assumes that all inputs in a transaction likely belong to the same owner. In a CoinJoin transaction all the outputs tend to be of the same amount. This makes it harder to define which input paid which output, somewhat breaking the absolute traceability of bitcoin transactions. As with any other anonymity network, a large and diverse group of participants will be more effective in disassociating the connections. CoinJoin transactions are not yet widely supported by bitcoin applications.
-
-Users still have to be mindful of how the UTXOs they received from the CoinJoin are spent. For instance, spending them together in a single transaction would unravel the anonymity gains from participating in the CoinJoin.
-
-</div>
-
-## Hiding Sensitive Information
-
-Imagine this scenario. The user is in a public place, and they need to make a payment using their bitcoin wallet. They open the wallet on their phone, but they don’t feel comfortable having their address and balance information clearly visible to strangers who may be looking over their shoulder, persons lurking, or video surveillance. Hence by giving users the ability to hide sensitive information in their wallet, but only when desired, they gain an added sense of physical privacy and security when using the app in public.
-
-#### What information is considered sensitive?
-
-Sensitive information in wallet applications include the wallet balance, addresses, private keys and previous transactions information.
-- Wallet Balance - shows how much is owned
-- Addresses - can be used to track on-chain transaction history
-- Invoices - can be used to track Lightning payment history
-- Private keys - can be used to access and transfer bitcoins
-
-It's more common for wallets to protect private keys, but not much is done for other sensitive information like the balance, addresses, and previous transactions. A few wallets like [Bitcoin Core](https://bitcoin.org/en/bitcoin-core/), [Wasabi](https://wasabiwallet.io/), [Muun](https://muun.com/), and others have made it work, though. Below are patterns and considerations for hiding and revealing sensitive information.
-
-### Quickly hiding balances
+#### Quickly hiding balances
 
 <div class="center" markdown="1">
```

Comment:
Why not just tap the balance to hide? Rather than add extra visuals with the eye icon. Could make it that hiding is a single tap and revealing is tap and hold (uses may accidently tap and reveal their balance when they don't want to.

Reply:
Both the balance and icon are tappable.

Tap and hold is a good addition. We'll need an indicator though that lets users know what to do. The first reaction is probably to assume single-tap to unhide because it only took a tap to hide.